programs/ziti-tunnel-app/netif_driver/linux/tun.c

```diff
@@ -100,5 +100,8 @@ netif_driver tun_open(char *error, size_t error_len) {
     driver->uv_poll_init = tun_uv_poll_init;
     driver->close        = tun_close;
 
+    system("ip link set tun0 up");
+    system("ip route add 169.254.0.0/16 dev tun0");
```

Comment:
You can use tun->name here instead of assuming "tun0".

Reply:
I'm guessing you intend to use `dns_block` for the route destination?

But the more I think about this, the more uncertain I am about adding the route here. What happens when the tunneler app doesn't have/use a dns manager?

Reply:
I think we have to use DNS manager all the time (most of the time?) otherwise we can't configure DNS intercepts, right?

Reply:
Most of the time, yes. But there are situations where the `--resolver none` option of the current ziti-tunnel for linux has been useful. Also consider at some point we may merge prox-c into the ziti-tunnel-c.

Reply:
added a condition to add route only if DNS block is set